File: PWP.InvoiceCapture.WebApplication/src/api/models/InvoiceManagement/InvoiceProcessingResult.ts
import { Invoice } from "./Invoice";
import { InvoiceDataAnnotation } from "./InvoiceDataAnnotation";

export interface InvoiceProcessingResult {
    id: number;
    invoiceId: number;
    createdDate: Date;
    modifiedDate: Date;
    templateId?: number;
    processingTypeId: number;
    dataAnnotationFileId: string;
    invoice: Invoice;
    dataAnnotation: InvoiceDataAnnotation;
    cultureName?: string;
    vendorName?: string;
}

File: PWP.InvoiceCapture.InvoiceManagement/Sources/PWP.InvoiceCapture.InvoiceManagement.Business.Contract/Models/InvoiceProcessingResult.cs
using PWP.InvoiceCapture.InvoiceManagement.Business.Contract.Enumerations;
using System;

namespace PWP.InvoiceCapture.InvoiceManagement.Business.Contract.Models
{
    public class InvoiceProcessingResult
    {
        public int Id { get; set; }
        public int InvoiceId { get; set; }
        public string VendorName { get; set; }
        public InvoiceProcessingType ProcessingType { get; set; }
        public string TemplateId { get; set; }
        public int TrainingFileCount { get; set; }
        public string DataAnnotationFileId { get; set; }
        public string InitialDataAnnotationFileId { get; set; }
        public string CultureName { get;set;}
        public DateTime CreatedDate { get; set; }
        public DateTime ModifiedDate { get; set; }
        public Invoice Invoice { get; set; }
        public DataAnnotation DataAnnotation { get; set; }
    }
}
